app/views/includes/head.html

```diff
@@ -1,6 +1,6 @@
 {% if fields.isGaEnabled.value === 'true' %}
     {% include "includes/scripts_ga_1.html" %}
 {% endif %}
-<link href="/public/stylesheets/application.css?v=4.2" media="screen" rel="stylesheet" type="text/css" />
+<link href="/public/stylesheets/application.css?v=4.4" media="screen" rel="stylesheet" type="text/css" />
```

Comment:
should be the same as avaya-webchat/template.html application.css?v=4.3?

Reply:
updated